printable-recipe-generator/src/tests/main.test.ts

```diff
@@ -3,11 +3,69 @@ import path from 'path';
 import * as ejs from 'ejs';
 import recipe from '../data/sampleRecipe.json';
 
+//load Contributors
+async function getChefs(): Promise<unknown> {
+	const resp = await fetch(
+		'https://recipes.guardianapis.com/v2/contributors.json',
+	);
+	try {
+		return resp.json().catch(console.error);
+	} catch (error) {
+		console.error('Failed to parse chefs JSON: ', error);
+		return null;
+	}
+}
+
+//load SVGs
+const svgPath = (fileName: string) =>
+	path.join(__dirname, '../assets/svgs/', fileName);
+
+const svgs = {
+	FeastLogo: fs.readFileSync(svgPath('FeastLogo.svg'), 'utf-8'),
+	TheGuardianLogo: fs.readFileSync(svgPath('TheGuardianLogo.svg'), 'utf-8'),
+	'dairy-free': fs.readFileSync(svgPath('Dairy-free.svg'), 'utf-8'),
+	'gluten-free': fs.readFileSync(svgPath('Gluten-free.svg'), 'utf-8'),
+	vegan: fs.readFileSync(svgPath('Vegan.svg'), 'utf-8'),
+	vegetarian: fs.readFileSync(svgPath('Vegetarian.svg'), 'utf-8'),
+	camera: fs.readFileSync(svgPath('camera.svg'), 'utf-8'),
+	'feast-book-outlined': fs.readFileSync(
+		svgPath('feast-book-outlined.svg'),
+		'utf-8',
+	),
+};
+
+//load fonts
+const fontPath = (fileName: string) =>
+	path.join(__dirname, '../assets/fonts/', fileName);
+const fontsBase64 = {
+	RegularEgyptianFont: fs
+		.readFileSync(fontPath('GuardianTextEgyptian-Regular.ttf'))
+		.toString('base64'),
+	RegularSansFont: fs
+		.readFileSync(fontPath('GuardianTextSans-Regular.ttf'))
+		.toString('base64'),
+	HeadlineSemiBoldFont: fs
+		.readFileSync(fontPath('GHGuardianHeadline-Semibold.otf'))
+		.toString('base64'),
+	SansBoldFont: fs
+		.readFileSync(fontPath('GuardianTextSans-Bold.ttf'))
+		.toString('base64'),
+};
+
```

Comment:
I would not duplicate these here. Rather I would put them into their own file - `assetloader.ts`? - and import them here. That way you know that you are getting the same code and you don't need to update it in multiple places.